model/src/main/kotlin/OrtResult.kt

```diff
@@ -102,6 +102,30 @@ data class OrtResult(
         result
     }
 
+    private data class PackageEntry(val curatedPackage: CuratedPackage, val isExcluded: Boolean)
+
+    private val packages: Map<Identifier, PackageEntry> by lazy {
```

Comment:
A KDoc would be helpful, explainng the benefit of this property.

Reply:
As discussed not mandatory fixing.